api/repositories/build_repository_test.go

```diff
@@ -511,6 +511,64 @@ var _ = Describe("BuildRepository", func() {
 			})
 		})
 	})
+
+	Describe("ListBuilds", func() {
+		var (
+			app1GUID     string
+			package1GUID string
+			namespace1   *korifiv1alpha1.CFSpace
+			cfOrg        *korifiv1alpha1.CFOrg
+			build1GUID   string
+			build1       *korifiv1alpha1.CFBuild
+			buildRecords []repositories.BuildRecord
+			fetchError   error
+		)
+
+		BeforeEach(func() {
+			build1GUID = uuid.NewString()
+			app1GUID = uuid.NewString()
+			package1GUID = uuid.NewString()
+			cfOrg = createOrgWithCleanup(ctx, prefixedGUID("org"))
+			namespace1 = createSpaceWithCleanup(ctx, cfOrg.Name, prefixedGUID("space2"))
+			build1 = &korifiv1alpha1.CFBuild{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      build1GUID,
+					Namespace: namespace1.Name,
+					Labels: map[string]string{
+						korifiv1alpha1.SpaceGUIDKey: namespace1.Name,
+					},
+				},
+				Spec: korifiv1alpha1.CFBuildSpec{
+					PackageRef: corev1.LocalObjectReference{
+						Name: package1GUID,
+					},
+					AppRef: corev1.LocalObjectReference{
+						Name: app1GUID,
+					},
+					Lifecycle: korifiv1alpha1.Lifecycle{
+						Type: "buildpack",
+						Data: korifiv1alpha1.LifecycleData{
+							Buildpacks: []string{},
+							Stack:      "",
+						},
+					},
+				},
+			}
+			Expect(k8sClient.Create(ctx, build1)).To(Succeed())
+		})
+
+		It("if user is a Space Developer it returns a list with one element", func() {
+			createRoleBinding(ctx, userName, spaceDeveloperRole.Name, namespace1.Name)
+			buildRecords, fetchError = buildRepo.ListBuilds(ctx, authInfo)
+			Expect(fetchError).NotTo(HaveOccurred())
+			Expect(buildRecords).To(HaveLen(1))
+		})
+		It("if user is not a Space Developer, it returns an empty array", func() {
```

Comment:
That `It` should be the default case which makes "if" in the description redundant. I.e. `It("returns an empty array"`, or `It("returns an empty array (as no roles assigned)"` if you prefer to make it clear

Reply:
adjusted accordingly